### PR TITLE
Fix onboarding dark theme and setup profile

### LINKED ISSUE #108: Fix: Dark theme Onboarding 1.1 - 1.2, Sign up & Sign in 3.16, 3.17
### Description

- [x] First screen after splash screen
- [x] Home screen
- [x] Your profile screen
- [x] Dark theme

### Description

<div style="display: flex; flex-wrap: wrap;">

</div>

### PR DESCRIPTION
**Pull request by mr3nz1**
### What does this pull request do?

Fixes design issues with and adds dark mode to the first, second screen and setup profile screens

### Description

- [x] First screen after splash screen
- [x] Home screen
- [x] Your profile screen
- [x] Dark theme

### Description

<div style="display: flex; flex-wrap: wrap;">

</div>

### CODE REVIEW

**Review by Paccyfic — APPROVED** [review]
great dark theme implementations..!!

**Review by Paccyfic — APPROVED** [review]
just noticed a small typing error on your PR's title..!!..😂😂

**Review by Irirwanirira — APPROVED** [review]
Good Job

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+EXPO_PUBLIC_API_URL=http://api.local
+EXPO_PUBLIC_API_KEY=abc123
```
Comment (Nkbtemmy) [inline .env]: Never push .env file; create .env.example to provide a reference

**Review by Nkbtemmy — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (mr3nz1) [reply .env]: Let  me make sure to

**Review by Nkbtemmy — APPROVED** [review]

**Pull request merged**